mkfs/main.c

```diff
@@ -440,6 +440,7 @@ static const char * const mkfs_usage[] = {
 	"Creation:",
 	OPTLINE("-b|--byte-count SIZE", "set size of each device to SIZE (filesystem size is sum of all device sizes)"),
 	OPTLINE("-r|--rootdir DIR", "copy files from DIR to the image root directory"),
+	OPTLINE("-u|--subvol SUBDIR", "create SUBDIR as subvolume rather than normal directory"),
```

Comment:
I didn't see any man page updates, as it would be much better to have more detailed requirements for the options (like SUBDIR must be a relative path inside ROOTDIR, etc).

Reply:
Thanks Qu. Pushed new version including man page change